app/src/main/java/io/intrepid/contest/screens/contestcreation/namecontest/NameContestContract.java

```diff
@@ -0,0 +1,4 @@
+package io.intrepid.contest.screens.contestcreation.namecontest;
+
+public class NameContestContract {
```

Comment:
I think this will change when you create presenters for your fragments, but did you mean not to use NameContestContract to abstract your Presenter/Activity?